xamarin/Xamarin.Forms
# Added IsOpen property in SwipeEndedEventArgs

### Description of Change ###

Based on feedback from emails, etc., added the `IsOpen` property in `SwipeEndedEventArgs`.
Using the SwipeEnded event was possible to detect when the swipe gesture was completed. However, we had no information about the status of the SwipeView, is it open or closed?.

### API Changes ###
 
Changed `SwipeEndedEventArgs` to add a new property:

`bool IsOpen { get; }`

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![swipeview-ended-isopen](https://user-images.githubusercontent.com/6755973/80360855-43a06d00-8880-11ea-954f-0e7a77348a98.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the SwipeView samples. Selected the "SwipeView Events" sample. Swipe to open the Swipe. Now, in the SwipeEnded event information can see if the SwipeView is open or not.

### PR Checklist ###
